docs/source/documentation.rst

```diff
@@ -365,6 +365,43 @@ This will generate **'axon_morphometrics.xlsx'** file in each of folders::
     ---- pixel_size_in_micrometer.txt
     ---- axon_morphometrics.xlsx
 
+Morphometrics file
+~~~~~~~~~~~~~~~~~~
+
+The resulting **'axon_morphometrics'** file will contain the following columns headings:
```

Comment:
```suggestion
The resulting **'axon_morphometrics'** file will contain the following columns headings. Most of the metrics are computed using `skimage.measure.regionprops <https://scikit-image.org/docs/stable/api/skimage.measure.html#regionprops>`_.
```
I would remove all the "Extracted with `skimage.measure.regionprops <https://scikit-image.org/docs/stable/api/skimage.measure.html#regionprops>`_." in the table (it appears almost at every row-- this duplication of information is unnecessary and distracting)-- and just put it there once.